set default graphics mode to text

#  This file is derived from https://github.com/opencomputeproject/onie/blob/master/patches/grub/e4a1fe391/set-default-graphics-mode-to-text.patch
Copyright (C) 2016 Pankaj Bansal <pankajbansal3073@gmail.com>

SPDX-License-Identifier:     GPL-2.0

For EFI set the default graphics mode to text.

This supresses an ugly, but harmless message about no video mode
found.

diff --git a/grub-core/loader/i386/linux.c b/grub-core/loader/i386/linux.c
index fddcc46..4598392 100644
--- a/grub-core/loader/i386/linux.c
+++ b/grub-core/loader/i386/linux.c
@@ -45,8 +45,8 @@ GRUB_MOD_LICENSE ("GPLv3+");
 #ifdef GRUB_MACHINE_EFI
 #include <grub/efi/efi.h>
 #define HAS_VGA_TEXT 0
-#define DEFAULT_VIDEO_MODE "auto"
-#define ACCEPTS_PURE_TEXT 0
+#define DEFAULT_VIDEO_MODE "text"
+#define ACCEPTS_PURE_TEXT 1
 #elif defined (GRUB_MACHINE_IEEE1275)
 #include <grub/ieee1275/ieee1275.h>
 #define HAS_VGA_TEXT 0
